gcloud/logging/_gax.py

```diff
@@ -73,6 +73,8 @@ def list_entries(self, projects, filter_='', order_by='',
                   if not None, indicates that more entries can be retrieved
                   with another call (pass that value as ``page_token``).
         """
+        if page_token is None:
+            page_token = INITIAL_PAGE
```

Comment:
@tseaver Should I add a note here and elsewhere to avoid the assumption that occurred in #2124 

i.e. the assumption of #2124 was that callers will either pass an explicit page token or won't pass one at all. But the failures make it clear that some callers pass an explicit `None` as page token